services/node_create.proto

```diff
@@ -27,62 +25,90 @@ option java_multiple_files = true;
-     * Multiple nodes can have the same account_id.
+    * A Node account identifier.
+    * <p>
+    * This field is REQUIRED.<br/>
+    * This account identifier MUST be in the "account number" form.<br/>
+    * This account identifier MUST NOT use the alias field.<br/>
+    * If the identified account does not exist, this transaction SHALL fail.<br/>
+    * Multiple nodes MAY share the same node account.
      */
-    AccountID account_id = 1;
+    proto.AccountID account_id = 1;
 
     /**
-     * Description of the node with UTF-8 encoding up to 100 bytes, optional field.
+     * A short description of the node.<br/>
+     * This value, if set, MUST NOT exceed 100 bytes when encoded as UTF-8.<br/>
+     * This value is OPTIONAL.
      */
     string description = 2;
 
     /**
-     * Ip address and port, mandatory field. Fully qualified domain name is
-     * not allowed here. Maximum number of these endpoints is 10.
-     * The first in the list is used as the Internal IP address in config.txt,
-     * the second in the list is used as the External IP address in config.txt,
-     * the rest of IP addresses are ignored for DAB phase 2.
+     * A list of service endpoints.<br/>
+     * These endpoints SHALL represent the published endpoints to which other
+     * consensus nodes may _gossip_ transactions.<br/>
+     * These endpoints MUST specify address and port.<br/>
+     * These endpoints MUST NOT specify a DNS name.<br/>
+     * This list MUST NOT be empty.<br/>
+     * This list MUST NOT contain more than `10` entries.<br/>
+     * The first entry in this list SHALL be the endpoint published to all
+     * consensus nodes.<br/>
+     * All other entries SHALL be reserved for future use.
      */
-    repeated ServiceEndpoint gossip_endpoint = 3;
+    repeated proto.ServiceEndpoint gossip_endpoint = 3;
 
     /**
-     * A node's grpc service IP addresses and ports, IP:Port is mandatory,
-     * fully qualified domain name is optional. Maximum number of these endpoints is 8.
+     * A list of service endpoints.<br/>
+     * These endpoints SHALL represent the published endpoints to which clients
+     * may submit transactions.<br/>
+     * These endpoints MUST specify address and port.<br/>
+     * These endpoints MUST NOT specify a DNS name.<br/>
+     * This list MUST NOT be empty.<br/>
+     * This list MUST NOT contain more than `8` entries.
      */
-    repeated ServiceEndpoint service_endpoint = 4;
+    repeated proto.ServiceEndpoint service_endpoint = 4;
 
     /**
-     * The node's X509 certificate used to sign stream files (e.g., record stream
-     * files). Precisely, this field is the DER encoding of gossip X509 certificate.
-     * This is a mandatory field.
+     * A certificate used to sign gossip events.<br/>
+     * This value MUST be a certificate of a type permitted for gossip
+     * signatures.<br/>
+     * The value presented MUST be a UTF-8 NFKD encoding of the TLS
+     * certificate.<br/>
```

Comment:
Keep this "Precisely, this field is the DER encoding of gossip X509 certificate."

Reply:
That sentence does not make sense.
Also, the latest update in channel is that we must use PEM encoding, not DER.
Note, that this won't merge before the next discussion where additional updates for the HIP are likely.

Reply:
Wording updated to flow better and be a bit more clear on formats and encoding.

Reply:
DER is the encoding format to store the certificate. PEM is the file extension.

Reply:
PEM is absolutely a format, and it's not identical to DER (although the two _can be_ combined).
DER is a binary format (ASN.1 "Distinguished Encoding Rules") for encoding data, often used for PKI certificates (of various forms).
PEM is an ASCII format (S/MIME "Privacy Enhanced Mail") that Base64 encodes binary data as ASCII text with specific delimiters.
When the two are combined (which is unambiguous here) as a "PEM encoded certificate" you get a PKI certificate encoded to binary and then Base64 encoded to ASCII text and stored in a file with specified delimiters.

Specifying DER here with a "PEM" file extension would leave us with an unusable binary file that cannot be properly processed as a PEM format file and might result in data corruption if the file is assumed to be actual PEM format and treated as 7-bit-safe ASCII.

Reply:
"Precisely, this field is the DER encoding of gossip X509 certificate." This was Nathan's words to keep during our team meeting together. We parse .PEM file and get X509Certificate and store it's getEncoded() which is DER.